linetools/lists/parse.py

```diff
@@ -912,51 +933,123 @@ def update_wrest(table, verbose=True):
     table['Ek'][mt[0]] = 52330.33 / u.cm
     '''
 #
+def load_datasets(datasets, tol=1e-3, use_cache=True):
```

Comment:
make tol lower? 1e-4?

Reply:
would require quite a bit of fussing.
Let's stick with 1e-3 for now.